tests/components/media_player/test_samsungtv.py

```diff
@@ -301,3 +306,66 @@ def test_turn_on(self):
         self.device._mac = "fake"
         self.device.turn_on()
         self.device._wol.send_magic_packet.assert_called_once_with("fake")
+
+
+@pytest.fixture
+def samsung_mock():
+    """Mock samsungctl."""
+    with patch.dict('sys.modules', {
+        'samsungctl': MagicMock(),
+    }):
+        yield
+
+
+@asyncio.coroutine
+async def test_play_media(hass, samsung_mock):
+    """Test for play_media."""
+    asyncio_sleep = asyncio.sleep
+    sleeps = []
+
+    async def sleep(duration, loop):
+        if duration > 0:
+            sleeps.append(duration)
+        await asyncio_sleep(0, loop=loop)
+
+    with patch('asyncio.sleep', new=sleep):
+        device = SamsungTVDevice(**WORKING_CONFIG)
+        device.hass = hass
+
+        hass.async_add_job = mock.Mock()
+        await device.async_play_media(MEDIA_TYPE_CHANNEL, "576")
+
+        send = device.send_key
+        expected = [(send, "KEY_5"), (send, "KEY_7"), (send, "KEY_7")]
+        hass.async_add_job == expected
```

Comment:
There's no `assert` here? Check the `call_args` of the mock instance. Use the `mock.call` helper when comparing with expected args.

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.call